src/main/java/edu/cornell/kfs/fp/businessobject/USBankRecordFieldUtils.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2016 The Kuali Foundation.
+ * 
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.opensource.org/licenses/ecl2.php
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.cornell.kfs.fp.businessobject;
+
+import java.io.IOException;
+import java.sql.Date;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+/**
+ * Provides methods for parsing USBank addendum records.
+ * In the long-run, it may be wise/useful to turn this into a subclass of 
+ * org.kuali.kfs.sys.businessobject.BusinessObjectStringParserFieldUtils and 
+ * provide subclasses that assist in parsing, but we won't for now.
+ *
+ */
+public class USBankRecordFieldUtils {
```

Comment:
Seems like this could be more generic, but I'm fine with leaving it specific for now and making it more generic later if/when that becomes beneficial.